src/theme/breakpoints.ts

```diff
@@ -0,0 +1,21 @@
+export const sm = 375
+export const md = 768
+export const lg = 1024
+export const xl = 1440
+
+export interface Breakpoints {
+  sm: string
+  md: string
+  lg: string
+  xl: string
+}
+
+export const breakpoints: string[] & Breakpoints = Object.assign(
+  [`${sm}px`, `${md}px`, `${lg}px`, `${xl}px`],
+  {
+    sm: `${sm}px`,
+    md: `${md}px`,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'md' is not sorted alphabetically](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4694377)